### PR TITLE
About page

### PR DESCRIPTION
**Pull request by belljustin**
refs #67

### CODE REVIEW

File: misirlou/urls.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
     url(r'^$', views.RootView.as_view(), name='api-root'),
-    url('^about/$', views.AboutView.as_view(), name='about'),
+    url('^about/$', views.RootView.as_view(), name='about'),
```
Comment (ahankinson) [inline misirlou/urls.py]: You can remove the second URL parameter and collapse it to a single matcher, I think.

`url(r'^$|^about/$', views.RootView.as_view(), name='api-root')`

File: misirlou/urls.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^$', views.RootView.as_view(), name='api-root'),
+    url(r'^$|^about/$', views.RootView.as_view(), name='api-root'),
```
Comment (ahankinson) [inline misirlou/urls.py:22]: Did you test this and see if it works?

Comment (belljustin) [inline misirlou/urls.py:22]: Yes

Comment (ahankinson) [inline misirlou/urls.py:22]: k

**Pull request merged**